pkg/cmd/config/config.go

```diff
@@ -0,0 +1,70 @@
+/*
+SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package config
+
+import (
+	"github.com/gardener/gardenctl-v2/internal/util"
+	"github.com/gardener/gardenctl-v2/pkg/cmd/base"
+
+	"github.com/spf13/cobra"
+)
+
+// NewCmdConfig returns a new config command.
+func NewCmdConfig(f util.Factory, o *ConfigOptions) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "config",
+		Short: "modify gardenctl configuration file using subcommands",
+		Long: `Modify gardenctl files using subcommands like \"gardenctl config set-garden my-garden\"
```

Comment:
```suggestion
		Long: `Modify gardenctl files using subcommands like "gardenctl config set-garden my-garden"
```
escape character not needed when using `